ffi/src/lib.rs

```diff
@@ -13,15 +13,23 @@ use std::sync::{Arc, RwLock};
 use firewood::db::{BatchOp as DbBatchOp, Db, DbConfig, DbViewSync as _, Proposal};
 use firewood::manager::{CacheReadStrategy, RevisionManagerConfig};
 
-mod metrics_setup;
-
 use metrics::counter;
 
+#[doc(hidden)]
+mod metrics_setup;
+
 #[global_allocator]
+#[doc(hidden)]
 static GLOBAL: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
 
+#[doc(hidden)]
```

Comment:
nit: I would keep this one, and add a doc comment. It's nice to see the type of our proposal IDs, especially since we return them (wrapped in a usize of course, but still).